config.md

```diff
@@ -428,7 +433,7 @@ Keys MUST be unique within this map.
 Keys MUST NOT be an empty string.
 Keys SHOULD be named using a reverse domain notation - e.g. `com.example.myKey`.
 Keys using the `org.opencontainers` namespace are reserved and MUST NOT be used by subsequent specifications.
-Implementations that are reading/processing this configuration file MUST NOT generate an error if they encounter an unknown annotation key.
+Runtimes MUST NOT generate an error if they encounter an unknown annotation key when reading/processing this configuration file.
```

Comment:
Suggest 'reading or processing' here and below.  I'm not a big fan of '/' over 'or'.  Again, my $.02, your call.